packages/blocks/src/components/WordPressBlocksProvider.tsx

```diff
@@ -20,6 +21,7 @@ export const WordPressBlocksContext =
 
 export interface WordPressBlocksContextType {
   blocks?: WordPressBlock[];
+  theme?: ThemeJson;
```

Comment:
I was wondering if you could create a new provider for the theme instead of using the 
`WordPressBlocksContextType` we would like to have this separate as much as possible.

Reply:
`WordPressBlocksThemeContext`

Reply:
Yeah that sounds good

Reply:
There are now two pieces of context that live inside `WordPressBlocksProvider`:

* `WordPressBlocksContext`
* `WordPressThemeContext`

Both of these contexts have been exported.

The values have also been changed to house just their specific pieces of context, for example `WordPressBlocksContext` only stores `blocks`. And `WordPressThemeContext` only stores the `theme`